.travis.yml

```diff
@@ -0,0 +1,109 @@
+    env: EXTRA_BUILD_FLAGS="-DUSE_SYSTEM_Boost=ON -DUSE_SYSTEM_SWIG=OFF -DUSE_SYSTEM_HDF5=OFF -DUSE_SYSTEM_FFTW3=OFF -DDEVEL_BUILD=ON" CC=gcc-5 CXX=g++-5
+  - os: osx
+    language: generic
+    env: EXTRA_BUILD_FLAGS="-DUSE_SYSTEM_HDF5=Off -DUSE_SYSTEM_SWIG=Off -DPYTHON_LIBRARY=$(python-config --prefix)/lib/libpython2.7.dylib -DPYTHON_INCLUDE_DIR=$(python-config --prefix)/include/python2.7" CC=gcc CXX=g++
+  - os: osx
+    language: generic
+    env: EXTRA_BUILD_FLAGS="-DUSE_SYSTEM_HDF5=OFF -DUSE_SYSTEM_SWIG=On -DDEVEL_BUILD=ON -DPYTHON_LIBRARY=$(python-config --prefix)/lib/libpython2.7.dylib -DPYTHON_INCLUDE_DIR=$(python-config --prefix)/include/python2.7 "
+  - os: osx
+    language: generic
+    env: EXTRA_BUILD_FLAGS="-DUSE_SYSTEM_SWIG=On -DUSE_SYSTEM_HDF5=Off -DUSE_SYSTEM_FFTW3=OFF -DDEVEL_BUILD=ON -DPYTHON_LIBRARY=$(python-config --prefix)/lib/libpython2.7.dylib -DPYTHON_INCLUDE_DIR=$(python-config --prefix)/include/python2.7" CC=gcc CXX=g++
+env:
+  global:
+    - BUILD_FLAGS="-DCMAKE_BUILD_TYPE=Release -DSIRF_TAG=$TRAVIS_COMMIT -DGADGETRON_TAG=e7eb430"
+
+# Ubuntu 14.04 LTS
+dist: trusty
+
+# No need for sudo
+sudo: false
+
+# Compilation dependencies
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - git-core
+      - build-essential
+      - g++-5
+      - libboost-all-dev
+      - libhdf5-serial-dev
+      - libfftw3-dev
+      - python-dev
+      - python3-dev
+      - liblapack-dev
+      - libxml2-dev
+      - libxslt-dev
+      - libarmadillo-dev
+      - libace-dev
+      - libgtest-dev
+      - libplplot-dev
+      - root-system-bin
+      - swig
+
+before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew tap homebrew/science; fi
+  - # Note: boost is already installed on osx on travis so should not be included. However, we currently need boost-python
+  - # if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install boost-python; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install ace; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install swig; fi
+  - #if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install scipy matplotlib; fi
+  - # need curl to get more recent cmake
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install curl; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then curl -0 https://cmake.org/files/v3.8/cmake-3.8.0-Darwin-x86_64.tar.gz -o cmake.tar.gz; fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then curl -0 https://cmake.org/files/v3.7/cmake-3.7.2-Linux-x86_64.tar.gz -o cmake.tar.gz; fi
+  - tar xzf cmake.tar.gz
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then mv cmake-*/CMake.app/Contents/* cmake-*; fi
+
+install:
+  - #pip install --upgrade pip setuptools wheel
+  - pip install --only-binary=numpy,scipy,matplotlib numpy scipy matplotlib
+  - pip install nose nose-timer codecov coveralls
+  - cd ..
+  - git clone https://github.com/CCPPETMR/SIRF-SuperBuild --recursive -b v0.9.2
```

Comment:
need to delete the 0.9.2 checkout and stick to master. (it will remove some errors)

Reply:
done